.github/workflows/ci.yml

```diff
@@ -0,0 +1,68 @@
+name: CI
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  BASENAME: "vorestation"
```

Comment:
This probably needs to be changed to "polaris" to make the checker work properly, but I don't quite have the experience (or setup) with Github CI to prove it.